src/components/layout/header/Header.tsx

```
@@ -2,6 +2,7 @@ import { useState, useEffect } from 'react';
 import { useTheme } from 'next-themes';
 import { FiSun, FiMoon } from "react-icons/fi";
 import AccountConnect from './AccountConnect';
+import { AGENT_NAME } from '@/utils/constants';
```

ESLint

[error] `@/utils/constants` import should occur before import of `./AccountConnect`

(import/order)